### PR TITLE
[docs] Add a limitation for CDC in docs

### PR DESCRIPTION
**Pull request by Sumukh-Phalgaonkar**
This PR adds a limitation to the list of limitationS for CDC.

### CODE REVIEW

**Review by vaibhav-yb — COMMENTED** [review]
LGTM. Adding @ddhodge from docs team for review.

File: docs/content/preview/explore/change-data-capture/using-logical-replication/_index.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ To understand how the various features and configuration of the connector, see [
 
 - CDC is not supported on a target table for xCluster replication [11829](https://github.com/yugabyte/yugabyte-db/issues/11829).
 
-- Currently we don't support schema evolution for changes that require table rewrites (ex: ALTER TYPE).
+- Currently we don't support schema evolution for changes that require table rewrites (ex: ALTER TYPE). Additionally the operations DROP TABLE and TRUNCATE TABLE are not supported by CDC.
```
Comment (ddhodge) [inline docs/content/preview/explore/change-data-capture/using-logical-replication/_index.md:108]:
```suggestion
- Currently, CDC doesn't support schema evolution for changes that require table rewrites (for example, ALTER TYPE), or DROP TABLE and TRUNCATE TABLE operations.
```

**Comment by Sumukh-Phalgaonkar** [comment]
> minor edit

Made the changes.

**Review by ddhodge — APPROVED** [review]

**Comment by Sumukh-Phalgaonkar** [comment]
@ddhodge @vaibhav-yb  I have added links to the ALTER TYPE page, in the point mentioning the CDC limitation about table rewrites. Can you PTAL, thanks.

File: docs/content/v2.20/explore/change-data-capture/cdc-overview.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ The core primitive of CDC is the _stream_. Streams can be enabled and disabled o
 * A single stream can only be used to stream data from one namespace only.
 * There should be a primary key on the table you want to stream the changes from.
 * CDC is not supported on a target table for xCluster replication [11829](https://github.com/yugabyte/yugabyte-db/issues/11829).
-* Currently we don't support schema evolution for changes that require table rewrites (ex: ALTER TYPE).
+* Currently, CDC doesn't support schema evolution for changes that require table rewrites (for example, [ALTER TYPE](../../../v2.20/api/ysql/the-sql-language/statements/ddl_alter_table#alter-type-with-table-rewrite)), or DROP TABLE and TRUNCATE TABLE operations.
```
Comment (ddhodge) [inline docs/content/v2.20/explore/change-data-capture/cdc-overview.md:50]:
```suggestion
* Currently, CDC doesn't support schema evolution for changes that require table rewrites (for example, [ALTER TYPE](../../../api/ysql/the-sql-language/statements/ddl_alter_table#alter-type-with-table-rewrite)), or DROP TABLE and TRUNCATE TABLE operations.
```

**Pull request merged**